client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-status-content/site-boost-column/index.tsx

```diff
@@ -84,27 +98,44 @@ export default function SiteBoostColumn( { site }: Props ) {
 
 	if ( hasBoost ) {
 		return (
-			<a
-				className="sites-overview__column-action-button is-link"
-				href={ jetpackBoostHref }
-				target="_blank"
-				rel="noreferrer"
-				onClick={ () => recordEvent( 'boost_column_configure_click' ) }
+			<span
+				className={
+					siteError
+						? 'sites-overview__disabled sites-overview__row-status'
+						: 'sites-overview__row-status'
+				}
 			>
-				{ translate( 'Configure Boost' ) }
-			</a>
+				<a
+					className="sites-overview__column-action-button is-link"
+					href={ siteError ? '' : jetpackBoostHref }
+					target="_blank"
+					rel="noreferrer"
+					onClick={ () => recordEvent( 'boost_column_configure_click' ) }
+				>
+					{ translate( 'Configure Boost' ) }
+				</a>
+			</span>
```

Comment:
This ideally needs testing by someone who is able to replicate the Configure Boost button here, as it may be possible to convert this to a button (and remove the ESlint ignore at the top of the file as a result).

Reply:
I've been able to replicate now and added a fix. 71897c5be417c8e374ee3e5010e3714237a3c1ee